src/chat/__tests__/fetchingReducer-test.js

```diff
@@ -97,4 +98,19 @@ describe('fetchingReducer', () => {
       expect(newState).toEqual(expectedState);
     });
   });
+
+  test('if fetched messages are from a search narrow, discard them', () => {
```

Comment:
This should be nested in the `describe('MESSAGE_FETCH_COMPLETE'` to make clearer the context of what it's testing.